### ISSUE TITLE
Table: caption and summary

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5234]
## Why would adopting this proposal be beneficial?
We could meet some customer suggestions.

## Context

https://www.w3.org/WAI/tutorials/tables/caption-summary/

_Captions and summaries provide information that can help users find, navigate, and understand tables. While they are not required in every case to meet WCAG, captions and summaries are fairly straightforward ways to provide such information that is often needed._

By helping to add subtitles we can encourage their use and improve overall accessibility.

Right now we can add a for example a <p> with Id and then add `aria-describedby` to table component, but `<caption>` make the description inside the table markup.

```html
<p id="example">This is a summary of the table</p>

<!-- This is the component with aria-describedby -->

<table aria-describedby="example">
</table>
```

**Comment by matuzalemsteles** [comment]
Just commenting on what we talked about in Slack, it is possible to add the `caption` as part of the Table composition, I'm going to use this issue to be able to add `caption` and `aria-describedby` in the examples.
